src/bin/rdcore/rootmap.rs

```diff
@@ -53,6 +53,12 @@ pub fn rootmap(config: &RootMapConfig) -> Result<()> {
     // systemd-fstab-generator, and it defaults to read-only otherwise
     kargs.push("rw".into());
 
+    let rootflags = runcmd_output!("coreos-rootflags")?;
```

Comment:
We could pass the path to the device node, in case the script wants to do some investigation.

Reply:
Good idea. Slight tweak on your suggestion: we pass it the root mountpoint instead of the device node.